schema.graphql

```diff
@@ -277,13 +277,29 @@ type CollectionStats {
     floorPrice: BigInt
     lastSale: BigInt
     highestSale: BigInt
+    supply: BigInt!
```

Comment:
supply is the sum(token.supply) needed to compute the percentage value of a trait.